modules/aws/ec2-files.go

```diff
@@ -221,18 +221,18 @@ func FetchFilesFromAsgsE(t *testing.T, awsRegion string, spec RemoteFileSpecific
 				return err
 			}
 
-			errorsOccurred := []error{}
-
 			for _, instanceID := range instanceIDs {
 				err = FetchFilesFromInstanceE(t, awsRegion, spec.SshUser, spec.KeyPair, instanceID, spec.UseSudo, curRemoteDir, spec.LocalDestinationDir, fileFilters)
 
 				if err != nil {
 					errorsOccurred = append(errorsOccurred, err)
+					err = nil // since we reuse err, make sure not to double add the same error if this loop exits
 				}
 			}
 
 			if err != nil {
 				errorsOccurred = append(errorsOccurred, err)
+				err = nil
 			}
```

Comment:
I'm actually not sure this whole block is necessary, given that you are checking `err` right after each call.

Reply:
Hmm - good point. I will remove this

Reply:
removed.